Sources/Hummingbird/Application.swift

```diff
@@ -54,7 +54,9 @@ public struct HBApplication: Sendable {
     // eventLoopGroup
     public let eventLoopGroup: EventLoopGroup
     // server
-    public let server: HBHTTPServer
+    internal let server: HBHTTPServer
+    // date cache service
+    public let dateCache: HBDateCache
```

Comment:
It's still `public`, is that intentional?

Reply:
damn I thought I changed that, fixed now